vtex/toolbelt
# Minor refactors and fix eventsources

#### What is the purpose of this pull request?

- [telemetry:collector] Flush just before exiting process.
- [telemetry:collector] Make flush sync (necessary to use `process.on('exit')`
- [sse] Check if token is valid before creating connection.
- [sse] Abort process on 401 or 403 server errors. 
- [sse:telemetry] Register sse errors with errorKind `SSEError`.
- [ErrorReport:truncateStrings] Use @vtex/api corrected version (stack overflow error in some rare cases)

#### How should this be manually tested?
```
cd /tmp && rm -rf toolbelt && \
git clone https://github.com/vtex/toolbelt.git && \
cd toolbelt && \
git checkout fix/debugger-eventsource && \
yarn && yarn watch
```
This is kinda difficult to test :(
I couldn't reproduce SSE errors, so I just made sure `vtex link` and `vtex login` were still working as expected.

#### Types of changes

#### Chores checklist
